reference/composedcubemaptexture.md

```diff
@@ -0,0 +1,126 @@
+## ComposedCubeMapTexture
+
+```
+ComposedCubeMapTexture {
+  vrmlField SFNode right   NULL
+  vrmlField SFNode left    NULL
+  vrmlField SFNode top     NULL
+  vrmlField SFNode bottom  NULL
+  vrmlField SFNode front   NULL
+  vrmlField SFNode back    NULL
+}
+```
+
+### Description
+
+The [ComposedCubeMapTexture](#composedcubemaptexture) node defines a cubic texture used for environment mapping. It is composed of exactly six [ImageTexture](imagetexture.md) nodes. If one of the [ImageTexture](imagetexture.md)s is not set, the corresponding side of the cubic texture will be black.
+
+Each of the six fields takes a texture corresponding to the side of the environment. For instance `right` takes the texture displayed on the side of the cube along the +X direction. Accordingly we have the following order: right: +X, left: -X, top: +Y, bottom: -Y, front: +Z, back: -Z.
+
+The [ComposedCubeMapTexture](#composedcubemaptexture) is typically used together with a [ComposedShader](composedshader.md). It can be set to an Appearance's texture field on its own, or in conjunction with other textures inside a [MultiTexture](multitexture.md) node.
```

Comment:
I think that the doc of this node should not refer to Appearance, but Appearance should explain how we can include a ComposedCubeMapTexture within (because Appearance uses the ComposedCubeMapTexture, not the inverse). Some modifications in reference/appearance.md are probably required to explain that ComposedCubeMapTexture is supported in Appearance.texture